modules/gltf/src/lib/gltf-scenegraph.js

```diff
@@ -487,7 +485,8 @@ export default class GLTFScenegraph {
     }
 
     // Update the glTF BIN CHUNK byte length
-    this.json.buffers[0] = {byteLength: totalByteLength};
+    this.json.buffers[0] = this.json.buffers[0] || {};
+    this.json.buffers[0].byteLength = totalByteLength;
```

Comment:
It is a little strange to just overwrite an externally provided buffer. Is it more a matter of setting default value?

```js
this.json.buffers[0].byteLength = this.json.buffers[0].byteLength || totalByteLength;
```

But I don't really have a clear picture of how an external buffer would be suppled so I am fine with this change for now.